includes/defaults.inc.php

```diff
@@ -599,7 +600,7 @@
 
 // Distributed Poller-Settings
 $config['distributed_poller']                            = false;
-$config['distributed_poller_name']                       = file_get_contents('/proc/sys/kernel/hostname');
+$config['distributed_poller_name']                       = trim(file_get_contents('/proc/sys/kernel/hostname'));
```

Comment:
I'm surprised they ever worked, because there's a newline in them; I noticed it when I was running the poller in debug mode.